src/Components/CurrentItem.js

```diff
@@ -0,0 +1,40 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { Item, Rating, Icon, Progress } from 'semantic-ui-react';
```

Comment:
I guess Item is something more React-like, but how about naming as `Games` or everything related to this? You are already using words like Rating, Progress, maybe is a better fit for semantic purposes.

Reply:
In this case, Item is a component directly from Semant-UI so changing the name here is not an option; the same applies for Rating and the rest that are imported from semantic-ui-react

Reply:
Oh yes, I didn't see Rating and Progress were imported from semantic-ui, I thought were custom. Ok, I understand.